tests/assignment/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -21,6 +21,33 @@ computedDescriptionLines =
 
 `;
 
+exports[`destructuring.js - flow-verify 1`] = `
+let {
+  bottom: offsetBottom,
+  left: offsetLeft,
+  right: offsetRight,
+  top: offsetTop,
+} = getPressRectOffset == null ? DEFAULT_PRESS_RECT : getPressRectOffset();
+
+const { accessibilityModule: FooAccessibilityModule, accessibilityModule: FooAccessibilityModule, accessibilityModule: FooAccessibilityModule, accessibilityModule: FooAccessibilityModule,
+      } = foo || {};
```

Comment:
I used to be printed as

```js
let {
  bottom: offsetBottom,
  left: offsetLeft,
  right: offsetRight,
  top: offsetTop
} =
  getPressRectOffset == null ? DEFAULT_PRESS_RECT : getPressRectOffset();
const {
  accessibilityModule: FooAccessibilityModule,
  accessibilityModule: FooAccessibilityModule,
  accessibilityModule: FooAccessibilityModule,
  accessibilityModule: FooAccessibilityModule
} =
  foo || {};
```

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAbOMAEwA6VOYBGEMMEAtkphAGY0DOGAQiWeQDR4Ho0xW0MMAGTi9O+TACcAlgHMAFn2p1GMAEpzF4gmQAO-FRgAqEXeIC+mALyZZGAAqS49emrhgYAeUNYrNqACuqKiYAPyYACIAogBiAIIAqkJGAPr2alEAypkpGQDCRphUdjCOzq7uXj4AFACUANxckFD0WMCYAIZgYOXShNKo0jAAngCyEAAmQXBUMRAQcd29-YMj41Po7J1LLn0DQ2OT07Pziz27Kwfr01td5-R7q4cbM5hzCzsPl2tHm9v3jyuv1e7zOy32Pxe2gIMMsNho80wAB8kdhzPUQOwQKYYNJoPRkKAOpJJBAAO72YkIAkoDoANwg0gmmJAhEkXQA1hhMroutIoLJkDBJAE4Fj+YxJKV2bJyB1kDQOqhGFiAFb0AAeTHZYC5MEyHXIcCE-LgCqVKpAvMkkuQrI6hGGqGgLN0MlgAHUmTB5MgABwABixbogjA97N0drdzjgkjpZqxTgAjgFpE5HB1ZfKkIrlWKQIxyNIhSL8w8BegAIoBEhmnMW-MwB1eiY+5AAJixwo6+wFeQocrtUGgCZAAUYRgdNNzjHM5iAA